packages/components/src/HopperProvider/docs/responsive-styles/mobile-first-do-not.tsx

```diff
@@ -0,0 +1,16 @@
+
+import { Div, Text } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
+        <Div
+            // This will only center text on screens 768px and wider, not on small screens
+            textAlign={{ sm: "left" }}
+            UNSAFE_width="30rem"
+            maxWidth="100%"
+            padding="inset-md"
+        >
+            <Text>Text Content</Text>
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/87b8b9f0-5dde-4149-a2da-708a4f0b4261)

Dark mode is broken

Reply:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/5caf4dd6-5a8f-4cee-9d35-7a7e7f785398)

I'm not sure it's clear how to see the code. could we see part of the code?